addon/components/au-checkbox-group.hbs

```diff
@@ -0,0 +1,18 @@
+<div
+  class="au-c-control-group {{this.alignmentClass}}"
+  role="group"
+  ...attributes
+  data-test-checkbox-group
+>
+  {{yield
+    (hash
+      Checkbox=(component
+        "au-checkbox"
+        inGroup=true
+        groupValue=@value
+        disabled=@disabled
+        onChangeGroup=@onChange
```

Comment:
Hmm, I'm not sure if I like this. The checkbox component now has to know about implementation details of the group.

I think it might be better to create a new private wrapper component that we yield instead. That component could contain the logic and then pass the normal `@onChange` and `@checked` arguments to the checkbox component. It's an extra layer but I think it will make the implementation more clear.

I poked at it in a separate PR and it seems to work.

It doesn't have to block the merging of this Pr though, we can change the implementation in a future PR.

Reply:
Seems ok to me 👍 Will the structure (in terms of usage) change with this PR?